원재/Week4_HelloObjWorld2.js

```diff
@@ -0,0 +1,52 @@
+const Person = {
+    name : 'name',
+    age : 'age',
+    stdNum : 'stdNum',
+    hobby : 'hobby',
+    grade : 'grade',
+    major : 'major',
```

Comment:
기본값을 이런식으로 정해주는건 딱히 권장하지 않아요
일반적으로는 string 타입의 경우 그냥 ''을 사용하여 초기값을 정해주는 편 입니다.
''로 설정해놓는다면 할당 오류로 값이 제대로 들어가지 않았을 때 체크하기가 쉬워지거든요!

```suggestion
    name : '',
    age : '',
    stdNum : '',
    hobby : '',
    grade : '',
    major : '',
```